1_ses_node_on_openstack/create.sh

```diff
@@ -27,7 +27,7 @@ openstack server add floating ip ${SERVER_NAME} $IP_CREATE
 
 pushd ${MAIN_FOLDER}
     echo ${IP_CREATE} > .ses_ip
-    grep ${IP_CREATE} inventory-ses.ini 2>&1 > /dev/null || (echo "${SERVER_NAME} ansible_ssh_host=${IP_CREATE} ansible_host=${IP_CREATE} ansible_user=root ansible_ssh_user=root" >> inventory-ses.ini)
+    grep ${IP_CREATE} inventory-ses.ini 2>&1 > /dev/null || (echo "[ses_nodes]" > inventory-ses.ini && echo "${SERVER_NAME} ansible_ssh_host=${IP_CREATE} ansible_host=${IP_CREATE} ansible_user=root ansible_ssh_user=root" >> inventory-ses.ini)
```

Comment:
Looks like ansible_ssh_* variables are deprecated ansible 2.x forward. May be we don't need to use it.

Also is assumption that there will be always one node SES deployment in automated deployments?

https://docs.ansible.com/ansible/2.4/intro_inventory.html#hosts-and-groups

Reply:
You're right, we should probably remove the `ansible_ssh_*` vars.

The step 1 deploys a ses_node_on_openstack, so it creates only one node (as it uses the ses-ansible AIO automation from flavio). Therefore it creates an inventory with only one node.

I guess we could re-create another "step 1" to create ses with multi node, and (should it be required) adapt step 2.